res_partner_operating_unit/security/res_partner_security.xml

```diff
@@ -4,17 +4,17 @@
     Copyright (C) 2019 Serpent Consulting Services
     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
 
-    <record id="ir_rule_res_partner_allowed_operating_units" model="ir.rule">
+     <record id="ir_rule_res_partner_allowed_operating_units" model="ir.rule">
         <field name="model_id" ref="base.model_res_partner" />
         <field
             name="domain_force"
         >[('operating_unit_ids', 'in', user.operating_unit_ids.ids)]</field>
```

Comment:
`[('operating_unit_ids', 'in', user.operating_unit_ids.ids)]` can be replaced with `[('operating_unit_ids', 'in', operating_unit_ids)]`